src/index.js

```diff
@@ -210,8 +210,9 @@ async function loader(source, inputSourceMap, overrides) {
         });
       }
 
-      const { code, map, metadata } = result;
+      const { code, map, metadata, externalDependencies } = result;
 
+      externalDependencies?.forEach(dep => this.addDependency(dep));
```

Comment:
Do we use optional chaining here because we can't control the return type of `overrides.result`? If so we might consider convert `babel-loader` to ts and publish typings.

Reply:
Yeah, I added `?` because current `overrides.result` functions probably don't return `externalDependencies`, if they replace the result object.